### ISSUE TITLE
cmake Error

### ISSUE THREAD

**Issue by pedrov98** [kenji-tojo/stealth-shaper#1]
Hi Kenji,

I'm a beginner at working with C++ and coding in general I was doing your installation steps one by one. But I am stuck with error, that (_stealth-shaper/stealth-shaper/ext/libigl does not contain a CMakeLists.txt file._)

The full output from terminal. I'm running it on Visual Studio Code on a Mac.
(base) pedrovasconcelos@Pedros-Air build % cmake -DCMAKE_BUILD_TYPE=Release ..                           
-- The CXX compiler identification is AppleClang 15.0.0.15000309
-- The C compiler identification is AppleClang 15.0.0.15000309
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at stealth-shaper/CMakeLists.txt:17 (add_subdirectory):
  The source directory

    /Users/pedrovasconcelos/Desktop/Programs/stealth-shaper/stealth-shaper/ext/libigl

  does not contain a CMakeLists.txt file.

-- Configuring incomplete, errors occurred!
(base) pedrovasconcelos@Pedros-Air build % 

Thank you for your help,

Kind regards,

Pedro

**Comment by kenji-tojo** [comment]
Hi Pedro,

Thank you for your interest!

Could you check if the directory stealth-shaper/stealth-shaper/ext/libigl contains any files?
If not, could you try cloning the stealth-shaper repository with --recursive option?

```
git clone --recursive https://github.com/kenji-tojo/stealth-shaper.git
```

Best,
Kenji

**Comment by pedrov98** [comment]
That was it thank you, Kenji! I had installed both libraries, but when cloning the folders were duplicated. 
Mine were like this:
stealth-shaper/stealth-shaper/ext/libigl/libigl
stealth-shaper/stealth-shaper/ext/bvh/bvh

I just moved them up a folder.

It's a really cool what you have written up I haven't been able to wrap my around how it works but it's really neat.. I'll try and have a bit fun with other shapes see how well it works ahaha.

Many thanks,

Pedro


The libraries for reference:
https://github.com/libigl/libigl
https://github.com/madmann91/bvh
